packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1486,7 +1488,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 await this.refreshLatestSummaryAck(
                     undefined,
                     version.id,
-                    new ChildLogger(summaryLogger, undefined, { safeSummary: true }),
```

Comment:
Should `ChildLogger`'s constructor be private?

Reply:
i do exactly that below!

Reply:
🧐